.rubocop.yml

```diff
@@ -30,12 +30,23 @@ Layout/EmptyLineAfterGuardClause:
 Layout/SpaceAroundEqualsInParameterDefault:
   Enabled: false
 
+Metrics/AbcSize:
```

Comment:
Typically we don't add these exceptions to the `.rubocop.yml` as we probably want to fix them in the future.  Instead, revert this change and do `rubocop --auto-gen-config` that'll put the exceptions you need into `.rubocop_todo.yml`

Reply:
Fixed.